### ISSUE TITLE
Git autorefresh not working if repository is in different partition

### ISSUE THREAD

**Issue by fcastilloec** [Microsoft/vscode#64775]
Issue Type: <b>Bug</b>

Steps to Reproduce:

1. Creat/clone a git repository in a different partition
2. Open folder in vscode
3. Make any change recognized by git (edit a file, add a file, etc.)

I should be able to see any changes on the "Source Control" sidebar. This doesn't occur if the repository is in the main partition where vscode and the system are installed.

Does this issue occur when all extensions are disabled?: Yes

I also want to point out that this has been reported in #42821 but that one is closed and there's no reply from mantainers. I openned a new one to drive attention to the problem in hope that it gets fixed.
Feel free to close this issue if #42821 gets reopened.

VS Code version: Code 1.29.1 (bc24f98b5f70467bc689abf41cc5550ca637088e, 2018-11-15T19:07:43.495Z) I also tested the insider version 1.30.0-1544221759 and it has the same error.
OS version: Ubuntu 18.04, kernel 4.15.0-42-generic

**Comment by joaomoreno** [comment]
Do you get file events on that partition? If you run with `code --verbose`, it should print out file events as they come.

**Comment by fcastilloec** [comment]
It seems I still do. Running with `--verbose`, I modified the README.md of a project and I saw these to messages:

`[15102:1212/123106.504611:INFO:CONSOLE(294)] "%c[IPC Library: File Watcher (chokidar)] %c[CHANGED] /home/fcastillo/workspace/electron-installer-redhat/README.md", source: file:///usr/share/code/resources/app/out/vs/workbench/workbench.main.js (294)`

`15102:1212/123106.557852:INFO:CONSOLE(294)] "%c[IPC Library: File Watcher (chokidar)] %c >> normalized  [CHANGED] /home/fcastillo/workspace/electron-installer-redhat/README.md", source: file:///usr/share/code/resources/app/out/vs/workbench/workbench.main.js (294)`

Let me know if you need anything else

**Comment by fcastilloec** [comment]
I just realized something else that's going to help with this issue. It's not the fact that the project is on a different partition/disk.
I have a `workspace` folder in a different disk (`/mnt/disk2/workspace`) that has all my projects, and I have a symlink of that folder in my home (`~/workspace`). When I access any project from `~/workspace`, autorefresh doesn't work; but if I open the project from it's original location `/mnt/disk2/workspace` then autorefresh works.

To further test that the problem was with symlinks and not partitions, I cloned `electron-quick-start` at `~/electron-quick-start` and created a symlink in `~/link-to-electron-quick-start`. I opened the project from `~/link-to-electron-quick-start` and autorefresh doesn't work, but openning from `~/electron-quick-start` does work.

I thought it was related to partitions, but it's just because my partitions are usually symlink to somewhere else.
